### PR TITLE
docs: side panel

### PR DESCRIPTION
**Pull request by nkrantz**
Documentation for the Side Panel component

### CODE REVIEW

File: packages/paste-website/src/pages/components/side-panel/index.mdx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+This wrapper controls the positioning of the Side Panel with relation to the page content and should always wrap both the Side Panel itslef as well as the main page content.
+
+### Side Panel Header
+
+The Side Panel Header is a container for the descriptive title of the panel and sometimes an associated icon, in addition to any action buttons used in the panel.
+
+### Side Panel Header Actions
+
+The Side Panel Header Actions component is a container for action buttons that are used in the Side Panel Header. A "close" button should always be included. Other common actions include a MoreIcon for additional options.
```
Comment (serifluous) [inline packages/paste-website/src/pages/components/side-panel/index.mdx:94]:
```suggestion
The Side Panel Header Actions component is a container for action buttons that are used in the Side Panel Header. A "close" button should always be included. Other common actions include a `MoreIcon` for additional options.
```

File: packages/paste-website/src/pages/components/side-panel/index.mdx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+- There should be a close button so screen readers have a specific close action to target.
+- There must be an accessible aria label.
+
+## Examples
+
+### Basic Side Panel
+
+// TODO: storybook example
+
+## Composition notes
```
Comment (serifluous) [inline packages/paste-website/src/pages/components/side-panel/index.mdx:80]: suggestion: add some document hierarchy to this section so it's clear what wraps what. Like, SidePanelHeader, SidePanelHeaderActions, and SidePanelBody should be bulletpoints or smaller headings under SidePanel

File: packages/paste-website/src/pages/components/side-panel/index.mdx [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+          <SidePanelButton variant="secondary">Probar Panel Lateral</SidePanelButton>
+        </Box>
+      </SidePanelPushContentWrapper>
+    </SidePanelContainer>
+  )`}
+/>
+
+## Composition notes
+
+The Side Panel comes with some smaller components that can be used to compose a Side Panel to your application's needs. All of the following components should be used inside of a Side Panel Container, with Side Panel and Side Panel Push Content Wrapper being its direct children. The Side Panel Container controls the positioning of the Side Panel with relation to the page content.
```
Comment (serifluous) [inline packages/paste-website/src/pages/components/side-panel/index.mdx:176]:
```suggestion
The Side Panel comes with some smaller components that can be used to compose a Side Panel to your application's needs. All of the following components should be used inside of a `SidePanelContainer`, with `SidePanel` and `SidePanelPushContentWrapper` being its direct children. The Side Panel Container controls the positioning of the Side Panel with relation to the page content.
```

Comment (serifluous) [reply packages/paste-website/src/pages/components/side-panel/index.mdx:176]: ^ just bc I first missed the part that it everything had to be wrapped with `SidePanelContainer`

**Review by serifluous — DISMISSED** [review]

**Review by serifluous — APPROVED** [review]

**Review by krisantrobus — APPROVED** [review]

**Pull request merged**